release/test_release_gold.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+source ../_liferay_common.sh
+source ../_test_common.sh
+source release_gold.sh --test
+
+function main {
+	set_up
+
+	test_not_update_release_info_date
+	test_update_release_info_date
+
+	tear_down
+}
+
+function set_up {
+	export _PROJECTS_DIR="${PWD}"/../..
+
+	if [ ! -d "${_PROJECTS_DIR}/liferay-portal-ee" ]
+	then
+		echo -e "The directory ${_PROJECTS_DIR}/liferay-portal-ee does not exist.\n"
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
```

Comment:
@kiwm, if you're not manipulating the return of this function in the function `main`, what is the purpose of return an error code?